isaac-udoh.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+	<title>HTML Exercise1</title>
+	<meta name="description" content="Welcome!">
+</head>
+<body>
+    <br>
+    <br>
+    <br>
+    
+    <img width= "600"src= "http://www.ambab.com/sites/default/files/blog/blog_custom/HTML5.png" alt="I am the future">
+    
+    <h1>
+        Of Patterns and Power: Web Standards Then and Now
+    </h1>
+	<p>
+        In 
+        <a href="http://v3.danielmall.com/articles/content-display-patterns/" target="_blank">"CONTENT Display Patterns"</a> 
+        (which all front-end folk should read), Dan Mall point to a truth not unlike the one 
+        <a href="https://24ways.org/2015/putting-my-patterns-through-their-paces/" target="_blank">Ethan Marcotte shared last month
+        </a> on<i> 24 ways</i>. It is a truth as old as standards-based design: Contruct your markup to properly support your content (not your design).
+		<br>
```

Comment:
Close your first `<p>` here, get rid of the `<br>s` and keep adding more paragraph tags.